db/data/20201112075313_remove_unassociated_facilities_from_production.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveUnassociatedFacilitiesFromProduction < ActiveRecord::Migration[5.2]
+  def up
+    return if SIMPLE_SERVER_ENV != "production"
+    return if ENV["DEFAULT_COUNTRY"] != "IN"
+
+    Facility.where(facility_group_id: nil).delete_all
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
How about a no-op log instead, so it doesn't get in the way if we need to reverse another migration?